manifests/init.pp

```diff
@@ -66,10 +83,19 @@
   class { '::shibboleth_idp::attribute_resolver': } ->
   class { '::shibboleth_idp::attribute_filter': }
 
+  class { '::shibboleth_idp::service':
+    service_name    => $service_name,
```

Comment:
Is there a reason to pass these params through? Why not just make the defaults use the params from the init class like all the other classes?